parquet-column/src/main/java/org/apache/parquet/column/statistics/BinaryStatistics.java

```diff
@@ -46,8 +46,8 @@ public void mergeStatisticsMinMax(Statistics stats) {
 
   @Override
   public void setMinMaxFromBytes(byte[] minBytes, byte[] maxBytes) {
-    max = Binary.fromByteArray(maxBytes);
-    min = Binary.fromByteArray(minBytes);
+    max = Binary.fromReusedByteArray(maxBytes);
+    min = Binary.fromReusedByteArray(minBytes);
```

Comment:
I should have asked this earlier, but, should there be a defensive copy here too?

We are assuming that minBytes / maxBytes will be re-used, so what's the lifecycle assumption being made? That these won't be modified until after the last call to updateStats is made? Is that a valid assumption? Seems like this should actually do a copy, right?